src/org/pentaho/di/trans/steps/cassandrasstableoutput/writer/CQL3SSTableWriter.java

```diff
@@ -0,0 +1,88 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2014 by Pentaho : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.pentaho.di.trans.steps.cassandrasstableoutput.writer;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import org.apache.cassandra.config.CFMetaData;
+import org.apache.cassandra.config.Schema;
+import org.apache.cassandra.io.sstable.CQLSSTableWriter;
+import org.pentaho.cassandra.CassandraUtils;
+import org.pentaho.di.core.row.RowMetaInterface;
+import org.pentaho.di.core.row.ValueMetaInterface;
+
+import com.google.common.base.Joiner;
+
+class CQL3SSTableWriter extends AbstractSSTableWriter {
```

Comment:
This class should have it's own unit test. You would need to setup for injection of the private instance variable and create them if they are not set. This way at unit test time, you can inject mocks and at runtime (or integration test time) it can do what it is currently doing to create them.